### PR TITLE
Change CreateJob to CreateWorker and move to sidekiq

### PR DESCRIPTION
**Pull request by luchiago**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Related Tickets & Documents
#5305 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

jobs

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @luchiago, can you split this in two? This job is called continuously and we shouldn't move it over in one go. 

Also I would raise the priority to medium if not high, what do you think @mstruve ?

ps. you can find out where this endpoint is called by grepping for `reaction-creation`:

```shell
➜  devto git:(master) rg reaction-creation
app/views/moderations/mod.html.erb
147:          .then(sendFetch('reaction-creation', formData))

app/assets/javascripts/utilities/sendFetch.js
31:    case 'reaction-creation':

app/assets/javascripts/initializers/initializeCommentsPage.js.erb
98:          .then(sendFetch('reaction-creation', formData))

app/assets/javascripts/initializers/initializeArticleReactions.js
85:    .then(sendFetch('reaction-creation', createFormdata()))

app/assets/javascripts/initializers/initNotifications.js
89:            .then(sendFetch('reaction-creation', formData))

app/assets/javascripts/initializers/initializeReadingListIcons.js
62:    .then(sendFetch('reaction-creation', buttonFormData(button)))
```

**Comment by mstruve** [comment]
I agree with @rhymes, lets split this into two PRs and bump the priority to high, these kinds of reactions people expect instant feedback on.  Thanks!

**Comment by luchiago** [comment]
Ok @mstruve @rhymes. I will do the changes and send them. Thanks for reviewing.

File: spec/workers/reactions/create_worker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+RSpec.describe Reactions::CreateWorker, type: :worker do
+  describe "#perform" do
+    let(:reactable_type) { "Comment" }
+    let(:user_id) { 8 }
+    let(:reactable_id) { 1 }
+    let(:category) { "like" }
+    let(:worker) { subject }
+
+    context " when no user found" do
```
Comment (mstruve) [inline spec/workers/reactions/create_worker_spec.rb]: Can you remove the leading space for each of these descriptions?

Comment (luchiago) [reply spec/workers/reactions/create_worker_spec.rb]: @mstruve Sure. I thought that was better for reading but I'll do the changes.

**Pull request merged**